.github/workflows/CI.yaml

```diff
@@ -295,3 +295,58 @@ jobs:
           version: "dev"
           klt-repo: ${{ github.workspace }}
           token: ${{ secrets.KEPTN_SANDBOX_BOT_TOKEN }}
+
+  helm_charts_build:
+    name: Publish helm chart changes to charts repo
+    if: github.event_name == 'push' && needs.prepare_ci_run.outputs.NON_FORKED_AND_NON_ROBOT_RUN == 'true'
+    needs: [ prepare_ci_run, build_image ]
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Check out klt repo
+      uses: actions/checkout@v3
+
+    - name: Check out helm-charts repo
+      uses: actions/checkout@v3
+      with:
+        repository: 'keptn/helm-charts'
+        path: ./helm-charts-repository
+        token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Set up Node
+      uses: actions/setup-node@v3.6.0
+      with:
+        node-version: 16
+
+    - name: Install readme generator
+      run: |
+        git clone https://github.com/bitnami-labs/readme-generator-for-helm.git
+        cd ./readme-generator-for-helm
+        npm ci
+        npm install --location=global ./
+        cd ..
```

Comment:
i think the docs process should be different, basically similar to what we have in keptn/keptn:
just have a checker pipeline that checks if the docs are up to date on PRs
and then we don't need to care about them anymore when moving the chart source over to the charts repo since everything was prepared already on PR level